impl/httpMetricsManager.go

```diff
@@ -2,24 +2,62 @@ package impl
 
 import (
 	"fmt"
+	"sort"
+	"strings"
 	"time"
 
+	"github.com/satori/go.uuid"
 	"github.com/xiaoyao1991/chukonu/core"
 )
 
 type HttpMetricsManager struct {
 	requestSent int
 	countQueue  chan int
+	records     HttpRecords
+	durationSum DurationSummation
+	numOfError  int
 	// throughputRecord []int
 }
 
+type HttpRecord struct {
```

Comment:
This struct is suppose to record some metadata about the resp/request, so I thinks it's best not to include some unnecessary field like response body in `ChukonuResponse`